tests/test_mock_sampler.py

```diff
@@ -60,7 +129,26 @@ def test_zephyr_topology(self):
                         tile_parameter*grid_parameter*(8*grid_parameter + 4))
         dit.assert_sampler_api(sampler)
         dit.assert_structured_api(sampler)
-    
+
+    def test_properties(self):
+        properties = {'topology' : {
+            'type' : 'pegasus',
+            'shape' : [5]}}
+        # Note, topology_type and topology_shape must be consistent
+        # or None.
+        with self.assertRaises(ValueError) as e:
+            sampler = MockDWaveSampler(topology_type='chimera',
+                                       properties=properties)
+        with self.assertRaises(ValueError) as e:
+            sampler = MockDWaveSampler(topology_shape=[4],
+                                       properties=properties)
+        sampler = MockDWaveSampler(properties=properties)
+        self.assertTrue(sampler.properties['topology']['type']=='pegasus')
```

Comment:
Here, and elsewhere, better to use specialized asserts, like [`assertEqual`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual) (but also `assertLess`, `assertAlmostEqual`, etc). That way you get a helpful error message on assertion failed (printed out LHS vs RHS), unlike with truthy assertion (where you get something like "assertion failed, False is not True").